README.md

```diff
@@ -1 +1,70 @@
-# angular-nested-views-readme
\ No newline at end of file
+# Nested views and dynamic URLs
+
+## Overview
+
+As mentioned earlier, `uiRouter` offers us nested views. These are extremely powerful - allowing us to have views inside views!
+
+## Objectives
+
+- Describe dynamic views
+- Write a dynamic route using ui-router
+- Assign relevant template
+
+## Nested View
+
+Imagine that we have normal setup for a web page - a header, a footer, and everything in between those change depending on what page we are on. This is great for most cases - we can swap between different pages such as home, about, contact, etc.
+
+But what if we want to have a settings page, with multiple different pages for us to look at? We might want to be able to edit profile information, but also change our notification preferences or even have a "delete your account page".
+
+We'd setup our settings route, and this would be displayed in our main section where all the routes go. What now? How do we get all these different pages rendered *inside* our settings page?
+
+We could just create a tabs component that changes what we can see depending on what tab has been clicked, but it means our URL will always stick at `/settings` rather than changing between `/settings/notifications` and `/settings/user`.
+
+This is where nested views come in. As mentioend before with states, a state can have multiple children states. These will then be rendered inside of our parent state - but we must make sure that we use the `ui-view` directive inside our parent state too.
+
+Let's setup our settings example in `uiRouter`.
+
+```js
```

Comment:
Show the code example with just the `settings` state first to let the student wrap their heads around it. Then, add in the nested state.